lms/routes.py

```diff
@@ -238,4 +238,5 @@ def includeme(config):  # pylint:disable=too-many-statements
         "/dashboard/organization/{public_id}/assignment/{id_}",
         factory="lms.resources.dashboard.DashboardResource",
     )
-    config.add_route("api.assignment.stats", "/api/assignment/{id_}/stats")
+    config.add_route("dashboard.api.assignment", "/api/assignment/{id_}")
```

Comment:
Namespacing all of this to dashboard. Yes it might be use outside that context in the future but we can worry about then.

Reply:
Shouldn't the paths also be prefixed with `/dashboard`?